src/components/Modal/Modal.js

```diff
@@ -0,0 +1,21 @@
+import styles from "./Modal.module.css";
```

Comment:
import {overlay, modal, modalContent, closeButton} from "./Modal.module.css";

Reply:
It is pure css so, it does not work like sass